src/style.css

```diff
@@ -29,9 +29,9 @@
 
 .label-search-icon {
   position: absolute;
-  top: 46%;
-  left: calc(var(--padding-left-big-search) / 2 + 1px);
-  transform: translateY(-30%) translateX(-30%);
+  top: calc(var(--big-search-height) / 2);
```

Comment:
I don't know how to write this item correctly

Reply:
This variable that i presented in the example needs to be created first in the root, similarly how you have done it for --padding-left-big-search